requirements.txt

```diff
@@ -1 +1,2 @@
 ladybug-core>=0.41.30
+honeybee-radiance-postprocess==0.4.272
```

Comment:
This creates a loop in the dependency tree because honeybee packages can depend on ladybug packages but not the other way around.

I know it seems redundant but I would rather this just be a dependnecy on nunpy and move/copy the `binary_to_array` function here to the intersection.py module.